CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
+
```

Comment:
Not related to this file.

Rename README.rst to README.md and update with the recommended paragraphs.

Reply:
Thanks, it's done in another PR!